ppl/zqd/postgres/config.go

```diff
@@ -0,0 +1,115 @@
+package postgres
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/go-pg/pg/v10"
+)
+
+type Config struct {
+	// Network type, either tcp or unix.
+	// Default is tcp.
+	Network string `json:"network,omitempty"`
+	// TCP host:port or Unix socket depending on Network.
+	Addr string `json:"addr"`
+
+	User      string `json:"user"`
+	Password  string `json:"password"`
+	Database  string `json:"database"`
+	TLSConfig *tls.Config
+}
```

Comment:
Could this just be `type Config struct { pg.Options }`?

Reply:
That's just crazy enough to work!